### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by msavy**
@EricWittmann @kahboom Do these changes seem reasonable? In particular the new bit about squashing commits.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,16 +40,16 @@ your local Maven repository, and runs all of the unit tests:
 The project can be built and deployed on a variety of runtime platforms, but if you want to see it in 
 action as quickly as possible try this:
 
-    $ mvn clean install -Pinstall-all-wildfly8
-    $ cd tools/server-all/target/wildfly-8.2.0.Final/
+    $ mvn clean install -Pinstall-all-wildfly9
```
Comment (EricWittmann) [inline README.md]: Should actually be wildfly10 now I think...

**Comment by EricWittmann** [comment]
Yes, I like the part about squashing commits.  I'll try to start pushing back on that more.

File: README.md (hunk 1 above)
Comment (msavy) [inline README.md]: I wasn't too sure about this, simply because of the relative URLs bug with KC. Or is that also in WF9?

Comment (msavy) [inline README.md]: @EricWittmann Happy to do either. Especially as we'll be upgrading to a version of KC with fixed relative URL handling.

Comment (EricWittmann) [inline README.md]: Yes, it should be 10.  The bug is with KC and has nothing to do with WF.  To fix the bug we need to upgrade to KC 1.9.

Comment (msavy) [inline README.md]: Yeah, I just thought we used a different version of KC with the WF9 vs WF10 and the version we were using with WF10 was the b0rked one. Anyway, I realise that's incorrect now :o).

Updated!

Comment (EricWittmann) [inline README.md]: The diff still shows WF 9 everywhere.  :)

Comment (msavy) [inline README.md]: Bash F5 a few times :o)

Comment (msavy) [inline README.md]: Well, it's updated in my repo (https://github.com/msavy/apiman/blob/readme-java8/README.md) but for some reason the change isn't reflecting here. Maybe I'll open a new PR.

**Pull request closed without merging**